apps/editors/helpers.py

```diff
@@ -220,6 +221,24 @@ def reviewers_score_bar(context, types=None, addon_type=None):
                                          addon_type=addon_type)))
 
 
+@register.inclusion_tag('editors/includes/files_view.html')
+@jinja2.contextfunction
+def all_distinct_files(context, version):
+    hashes_to_file = {}
```

Comment:
Add a comment above this line with an example of what the structure will look like